nilearn/surface/surface.py

```diff
@@ -968,6 +968,44 @@ def _gifti_img_to_mesh(gifti_img):
     return coords, faces
 
 
+def combine_img_hemispheres(mesh):
```

Comment:
```suggestion
def combine_hemispheres_meshes(mesh):
```

When I read the function I did not realize you were going to merge only the meshes.

Reply:
done!